oxidauth-postgres/src/users/select_users_by_ids_query/mod.rs

```diff
@@ -0,0 +1,67 @@
+use oxidauth_repository::users::select_users_by_ids_query::*;
+use sqlx::PgConnection;
+
+use crate::prelude::*;
+
+use super::{TryFromUserRowError, UserRow};
+
+#[async_trait]
+impl<'a> Service<&'a FindUsersByIds> for Database {
+    type Response = (Vec<User>, Vec<Uuid>);
```

Comment:
Let's make a named struct for this instead of just a tuple -- should make it easier to understand as we move it around.